### PR TITLE
mslink: init at 1.3

### PR DESCRIPTION
**Pull request by mkg20001**
###### Motivation for this change

Add new package

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/mslink/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  '';
+
+  installPhase = ''
+    install -D mslink $out/bin/mslink
+  '';
+
+  meta = with lib; {
+    description = "Create Windows Shortcut Files (.LNK) without using Windows";
+    homepage = "http://www.mamachine.org/mslink/index.en.html";
+    license = licenses.gpl3Only;
```
Comment (fabaff) [inline pkgs/tools/misc/mslink/default.nix:26]:
```suggestion
    license = licenses.gpl3Plus;
```

see mslink.c file header "...either version 3 of the License, or (at your option) any later version."

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114281` at 2e5fa27d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/14d9036f6e7786759349fd3f9ba437f8">mslink</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/misc/mslink/default.nix:19:3:
```
   |
19 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/mslink/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+, lib
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mslink";
+  version = "1.3";
+
+  src = fetchurl {
+    url = "http://www.mamachine.org/mslink/mslink_v${version}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/mslink/default.nix:11]: Please use https.

File: pkgs/tools/misc/mslink/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    sha256 = "1qiwqa6w2in6gk4sxiy37c2wwpakin6l2ad2cf5s7ij96z2ijgqg";
+  };
+
+  preBuild = ''
+    rm mslink # clean up shipped executable
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/mslink/default.nix:16]: Please move this into postPatch.

**Pull request merged**